template/.github/workflows/ios_deploy_to_testflight.yml

```diff
@@ -25,10 +25,11 @@ jobs:
       with:
         ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
 
-    - uses: subosito/flutter-action@v1
+    - name: Setup Flutter environment
+      uses: subosito/flutter-action@v2
       with:
         channel: 'stable'
-        flutter-version: '2.10.3'
+        flutter-version: '3.3.10'
```

Comment:
Please update at this [README](https://github.com/nimblehq/flutter_templates/tree/develop/.template) as well 🙏 
![Screenshot 2023-03-01 at 09 30 54](https://user-images.githubusercontent.com/17875522/222029982-d4dd175a-4f1f-415e-b589-78b5ceaad864.png)

Reply:
@markgravity I think the Flutter version in CI/CD is another story. As we still define the Flutter SDK version as [`>=2.17.5 <3.0.0`](https://github.com/nimblehq/flutter_templates/blob/develop/.template/pubspec.yaml#L21). We don't have to force using `3.3.10`.